structs/student_validator.c

```diff
@@ -0,0 +1,46 @@
+#include<stdio.h>
```

Comment:
Get into a habit of putting space between `e` and `<` in the includes. 
```suggestion
#include <stdio.h>
```